GitUI/CommandsDialogs/SettingsDialog/Pages/BlameViewerSettingsPage.cs

```diff
@@ -51,6 +51,11 @@ protected override void PageToSettings()
             base.PageToSettings();
         }
 
+        private void cbDetectMoveAndCopyInThisFile_InfoClicked(object sender, EventArgs e)
```

Comment:
We should use `UserManual.UserManual.UrlFor(ManualSectionSubfolder` here as well instead of hardcoding the url.